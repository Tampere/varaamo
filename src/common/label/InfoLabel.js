import React from 'react';
import PropTypes from 'prop-types';
import Label from 'react-bootstrap/lib/Label';

const InfoLabel = ({ labelStyle, labelText }) => {
  return (
    <div className="app-InfoLabel">
      <Label bsStyle={labelStyle}>
        { labelText }
      </Label>
    </div>
  );
};

InfoLabel.propTypes = {
  labelStyle: PropTypes.string.isRequired,
  labelText: PropTypes.string.isRequired,
};

export default InfoLabel;
