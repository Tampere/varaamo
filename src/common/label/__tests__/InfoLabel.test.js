import React from 'react';
import toJson from 'enzyme-to-json';
import { shallow } from 'enzyme';

import InfoLabel from '../InfoLabel';

describe('InfoLabel', () => {
  test('renders correctly', () => {
    const labelStyle = 'success';
    const labelText = 'Success';
    const wrapper = shallow(
      <InfoLabel labelStyle={labelStyle} labelText={labelText} />,
    );

    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
