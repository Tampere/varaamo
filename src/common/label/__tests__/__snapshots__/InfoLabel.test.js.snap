// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`InfoLabel renders correctly 1`] = `
<div
  className="app-InfoLabel"
>
  <Label
    bsClass="label"
    bsStyle="success"
  >
    Success
  </Label>
</div>
`;
